app/controllers/people_controller.rb

```diff
@@ -6,19 +6,19 @@ class PeopleController < ApplicationController
   def signin
     #This is the sign-in sheet, not anything about authentication
     @people = Person.active
-    @department = "Police"
+    @department = Department.where(name: "Police")&.first
     @page_title = "Sign-in"
     render :layout => "print_signin"
   end
 
   def orgchart
-    @people = Person.police.active
+    @people = Person.active.select{|person| person.department&.name == "Police"}
     @page_title = "Org Chart"
     render :layout => "orgchart"
   end
 
   def police
-    @people = Person.police.active
+    @people = Person.active.select{|person| person.department&.name == "Police"}
```

Comment:
We have so many hard-coded references to Police throughout the app that I am wondering whether the concept of a Police Department is core domain knowledge.  If it's unavoidable to have that department, then we should perhaps create a Police subclass of Department, or else put a class method on Department to return the "Police" Department and a scope on Person to return people belonging to that department.